README.md

```diff
@@ -798,6 +798,12 @@ Variants:
 - Running on OpenShift with a stable Red Hat SSO
 - Running on OpenShift with the latest Red Hat SSO
 
+### `security/permissions`
+
+Verifies usage of `@PermissionAllowed` and `@PermissionAllowed` defined via meta-annotation.
```

Comment:
```suggestion
Verifies direct usage of `@PermissionAllowed` and `@PermissionAllowed` defined via meta-annotation.
```

Reply:
but don't push change just because of that (unless I find something more or Jose has request for changes) because I want CI to pass and make backports